app/src/components/group_schema_table.rs

```diff
@@ -154,7 +155,35 @@ impl GroupSchemaTable {
         let hardcoded = ctx.props().hardcoded;
         html! {
             <tr key={attribute.name.clone()}>
-                <td>{&attribute.name}</td>
+                {
+                    if hardcoded {
+                        let desc = group::resolve_group_attribute_description_or_default(&attribute.name);
+                        html!{
+                            <td>
+                                {desc.attribute_name.clone()}
```

Comment:
Do you need the clone? I would expect that a reference should be emough

Reply:
Probably not, I'll fix it.